### PR TITLE
Add dri for OpenGL

### PR DESCRIPTION
**Pull request by Justinzobel**
This is required if the user wants to do 3D Accelerated VMs

### CODE REVIEW

**Comment by Justinzobel** [comment]
This should fix #77 and #97

File: org.gnome.Boxes.json [hunk 1]
```diff
@@ -18,6 +18,7 @@
         "--socket=pulseaudio",
         "--share=network",
         "--device=all",
+        "--device=dri",
```
Comment (felipeborges) [inline org.gnome.Boxes.json:21]: Shouldn't `device=all` already cover `dri` and other all devices?

Comment (A6GibKm) [reply org.gnome.Boxes.json:21]: yes it does. Some repositories have it explicitly though.

Comment (Justinzobel) [reply org.gnome.Boxes.json:21]: 3D Acceleration didn't work for me until I added `dri`.

Comment (AsciiWolf) [reply org.gnome.Boxes.json:21]: That seems to be some weird Flatpak bug on your side. All my Flatpaks that use `device=all` have working OpenGL/Vulkan support.

**Comment by Justinzobel** [comment]
I don't use it any more so I'm not fussed about this now. If anyone else wants to take over if they see the same issue feel free. But for now I'm going to close it and unsubscribe.

**Pull request closed without merging**